test/utils.jl

```diff
@@ -10,7 +10,7 @@ r = RealInterval(1.5, 4.0)
 
 @test partype(Gamma(1, 2)) == Float64
 @test partype(Gamma(1.1, 2)) == Float64
-@test partype(Normal(1//1, 2//1)) == Rational{Int}
+@test partype(Normal(1//1, 2//1)) == Float64
```

Comment:
Ah, we have a constraint: the mean and the eltype of the distribution need to agree. Makes sense for a Normal, it can neither be sampled nor the pdf evaluated in Rational.

Reply:
Worth adding that as a comment?